resources/alma.rb

```diff
@@ -0,0 +1,62 @@
+resource_name :osl_repos_alma
+provides :osl_repos_alma
+unified_mode true
+
+default_action :add
+
+# These properties indicate whether or not a repo should be enabled
+property :base, [true, false], default: true
+property :extras, [true, false], default: true
+property :appstream, [true, false], default: true
+property :powertools, [true, false], default: true
+property :highavailability, [true, false], default: false
+property :exclude, Array, default: []
+
+action :add do
+  # Manage components of the main yum configuration file.
+  yum_globalconfig '/etc/yum.conf' do
+    cachedir '/var/cache/dnf'
+    installonly_limit '2'
+    installonlypkgs 'kernel kernel-osuosl'
+    clean_requirements_on_remove true
+  end
+
+  passthrough = {
+    'exclude' => (new_resource.exclude.join(' ') unless new_resource.exclude.empty?),
+  }.compact
+
+  yum_alma_baseos 'default' do
+    mirrorlist nil
+    baseurl "#{alma_url}/#{release_var}/BaseOS/#{base_arch}/os/"
+    passthrough passthrough
```

Comment:
This is generally bad practice and could make some debugging an issue later on. Please rename the variable to something else.